src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
 // write your code here
+const rows = document.querySelectorAll('table tr');
+
+for (const i of rows) {
+  const lastChild = i.querySelector(':last-child');
+  const secondChild = i.querySelector(':nth-child(2)');
+  const copySecondchild = document.createElement(secondChild.tagName);
```

Comment:
Ensure that `secondChild` is not null before trying to access its `tagName`. If a row has fewer than two children, `secondChild` will be null, leading to an error.